plugins/modules/aws_s3.py

```diff
@@ -156,6 +156,18 @@
     description:
       - KMS key id to use when encrypting objects using I(encrypting=aws:kms). Ignored if I(encryption) is not C(aws:kms).
     type: str
+  tags:
+    description:
+      - Tags dict to apply to the S3 object.
+    type: dict
+    version_added: 1.5.0
+  purge_tags:
+    description:
+      - Whether or not to remove tags assigned to the DB cluster if not specified in the playbook.
```

Comment:
```suggestion
      - Whether or not to remove tags assigned to the S3 object if not specified in the playbook.
```

Reply:
> This looks good, for `put` modes I think we should just call `ensure_tags()` after PUTing the object, rather than do a bunch of data transformation to provide the tags as ` Tagging='key1=value1&key2=value2',`

Thank you for reviewing @jillr. Ok, I was also thinking to do it in this way.